packages/core/app/fs-next-update/page.tsx

```diff
@@ -1,3 +1,63 @@
-export default function Page() {
-  return <div>placeholder</div>
+import CUSTOM_COMPONENTS from 'src/customizations/src/components'
+import type { PageContentType } from 'src/server/cms'
+import { getPage } from 'src/server/cms'
```

Comment:
Not blocking: what about migrating to the app directory already?

Reply:
Done